src/game_objects/io_queue.py

```diff
@@ -1,19 +1,25 @@
 from queue import SimpleQueue
 from random import randint
 
+from lib import event_manager
 from lib.game_object import GameObject
 from lib.game_event_type import GameEventType
 from game_objects.views.io_queue_view import IoQueueView
 
 
 class IoQueue(GameObject):
+
+    Instance = None
```

Comment:
I am not fond of `Instances` , `Pages`  and `Processes` being directly in the classes. Could these be held elsewhere?

Reply:
It makes sense, I believe both ProcessManager and PageManager can hold these `dict`s